### PR TITLE
Add Color to Force Directed Graph

### PR DESCRIPTION
**Pull request by ddol**
Issue: #1991 

- Adds Color to the Force Directed Graph. 
- Uses the d3 category20() scale. 
- Updates to the sample `energy.json` dataset with a `cluster` field denoting Agriculture/Industry.

screen shot 2017-01-17 at 01 01 53

@mistercrunch

### CODE REVIEW

**Comment by mistercrunch** [comment]
![img] (http://bestanimations.com/Animals/Mammals/Dolphins/animated-dolphin-gif-20.gif)

File: superset/assets/visualizations/directed_force.js [hunk 1]
```diff
@@ -157,9 +158,14 @@ function directedForceVis(slice) {
...
       .range([3, 30]);
 
+      const color = d3.scale.category20();
+
       node.append('circle')
       .attr('r', function (d) {
         return circleScale(Math.sqrt(d.total));
+      })
+      .style('fill', function (d) {
+        return color(d.color);
```
Comment (mistercrunch) [inline superset/assets/visualizations/directed_force.js]: will that change the default color?

Comment (ddol) [inline superset/assets/visualizations/directed_force.js]: Yes, from gray to blue.

**Comment by ddol** [comment]
I incorporated the rebase from #2005 . PTAL

**Comment by mistercrunch** [comment]
Closing old PRs with broken build or merge conflict, feel free to re-open once merge conflicts are sorted out.

**Comment by pirate** [comment]
any chance of this getting reopened and merged someday? I'd love to see this added

**Pull request closed without merging**